replications/internal/queue_management_test.go

```diff
@@ -437,6 +437,82 @@ func TestEnqueueData(t *testing.T) {
 	require.Equal(t, data, string(written))
 }
 
+// this test ensures that data does not get incorrectly dropped from the Queue on remote write failures
+func TestSendWrite(t *testing.T) {
```

Comment:
Did this test fail for the old code?

Reply:
No, but was a test I wanted to make sure passed since I wasn't sure of the scale of the original issue this PR fixes